web/libs/editor/src/components/App/App.jsx

```diff
@@ -219,6 +226,7 @@ class App extends Component {
       </Block>
     );
 
+    const shouldShowPanels = isDefined(store) && (!isFF(FF_BULK_ANNOTATION) || !store.hasInterface("annotation:bulk"));
```

Comment:
`store` is definitely defined at this point. and I'd suggest to rename it to `isBulkAnnotation` so later in the code it will be more clear when we display panels and when we don't

Reply:
It's kinda depends on what you want to make clear, but I'm ok with that change.